### PR TITLE
Minor restructure and update to the root and examples

### PR DESCRIPTION
**Pull request by haipinglu**
### Description
- Move code of conduct and contributing guidelines to `.github` following the latest [vissl](https://github.com/facebookresearch/vissl/) from facebookresearch.
- Remove data folder under examples and move the readme to the example readme. 
- Minor update to readme.

### Status
**Ready**

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -73,7 +73,7 @@ We appreciate all contributions. You can contribute in three ways:
 - Use PyKale in your project and let us know any bugs (& fixes) and feature requests/suggestions via creating an [issue](https://github.com/pykale/pykale/issues).
 - Contribute your code such as new features, functions, or examples via [fork and pull](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/about-collaborative-development-models) to become one of the [contributors](https://github.com/pykale/pykale/graphs/contributors).
 
-See [contributing guidelines](https://github.com/pykale/pykale/blob/master/CONTRIBUTING.md) for more details. You can also reach us via <a href="mailto:pykale-group&#64;sheffield.ac.uk">email</a> if needed. The participation in this open source project is subject to [Code of Conduct](https://github.com/pykale/pykale/blob/master/CODE_OF_CONDUCT.md).
+See [contributing guidelines](https://github.com/pykale/pykale/tree/master/.github/CONTRIBUTING.md) for more details. You can also reach us via <a href="mailto:pykale-group&#64;sheffield.ac.uk">email</a> if needed. The participation in this open source project is subject to [Code of Conduct](https://github.com/pykale/pykale/blob/master/CODE_OF_CONDUCT.md).
```
Comment (bobturneruk) [inline README.md:76]: Is the code of conduct link correct?

File: examples/README.md [hunk 2]
```diff
@@ -1,10 +1,18 @@
...
+Demos to show key functionalities via notebooks and GUI applications, with [TorchScript](https://pytorch.org/docs/stable/jit.html) support (in future).
 
 Name convention: `data_method` or `data_method_lightn` for lightning
 
-Domain-specific development in three areas
+## Separate data and code
+
+To download by code or other ways to local directory rather than upload to GitHub.
+
+In general, we do not upload data to keep the repository size small. All data in examples (so far) are from the public domain so they are downloaded into local directories rather than uploaded to GitHub. This is done by setting `.gitignore` (see line 7). If we do want to share data, the data should be external to the PyKale repository (Google drive is popular nowadays).
```
Comment (bobturneruk) [inline examples/README.md:11]: Maybe say that instructions for getting data are with the specific examples? I guess we may want to think about consistent ways of sharing data in future, if there is data that is essential for pykale to function. It's nice to have DOIs rather than links to commercial cloud storage as these have greater longevity. But don't let this slow down a merge.

File: README.md [hunk 3]
```diff
@@ -73,7 +73,7 @@ We appreciate all contributions. You can contribute in three ways:
 - Use PyKale in your project and let us know any bugs (& fixes) and feature requests/suggestions via creating an [issue](https://github.com/pykale/pykale/issues).
 - Contribute your code such as new features, functions, or examples via [fork and pull](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/about-collaborative-development-models) to become one of the [contributors](https://github.com/pykale/pykale/graphs/contributors).
```
Comment (bobturneruk) [inline README.md:74]: Not part of this PR, but do we prefer branches? Better to refer people to contributing guidelines for this?

**Review by bobturneruk — APPROVED** [review]
Looking good! Few comments - maybe a broken link, but otherwise these are considerations for the future.

**Comment by haipinglu** [comment]
> Looking good! Few comments - maybe a broken link, but otherwise these are considerations for the future.

Many thanks. Further changes are made to address your comments.

**Comment by haipinglu** [comment]
> Looking good! Few comments - maybe a broken link, but otherwise these are considerations for the future.

Many thanks. Further changes are made to address your comments.

**Comment by haipinglu** [comment]
> Looking good! Few comments - maybe a broken link, but otherwise these are considerations for the future.

Many thanks. Further changes are made to address your comments.
